cpp/src/groupby/sort/group_rank_scan.cu

```diff
@@ -115,13 +118,47 @@ std::unique_ptr<column> dense_rank_scan(column_view const& order_by,
     order_by,
     group_labels,
     group_offsets,
-    [] __device__(bool equality, auto row_index) { return equality; },
+    [] __device__(bool unequal, auto row_index_in_group) { return unequal ? 1 : 0; },
     DeviceSum{},
     has_nested_nulls(table_view{{order_by}}),
     stream,
     mr);
 }
 
+std::unique_ptr<column> percent_rank_scan(column_view const& order_by,
+                                          device_span<size_type const> group_labels,
+                                          device_span<size_type const> group_offsets,
+                                          rmm::cuda_stream_view stream,
+                                          rmm::mr::device_memory_resource* mr)
+{
+  auto const rank_column = rank_scan(
+    order_by, group_labels, group_offsets, stream, rmm::mr::get_current_device_resource());
+  auto const rank            = rank_column->view();
```

Comment:
Suggest to rename this as it is confusing.
```suggestion
  auto const rank_view            = rank_column->view();
```

Reply:
Fixed.